yarn-project/simulator/src/public/executor.ts

```diff
@@ -55,14 +70,20 @@ export class PublicExecutor {
     PublicExecutor.log.verbose(`[AVM] Executing public external function ${fnName}@${address}.`);
     const timer = new Timer();
 
-    const trace = new PublicSideEffectTrace(startSideEffectCounter);
+    const innerCallTrace = new PublicSideEffectTrace(startSideEffectCounter);
```

Comment:
It would be maybe good to add a comment here explaining what these traces are. But only if you need to change anything else.